src/scss/pages.scss

```diff
@@ -123,6 +123,62 @@
     }
 }
 
+#sso-page {
+    .content {
+        width: 300px;
+
+        .box {
+            margin-top: 30px;
+            margin-bottom: 30px;
+            text-align: center;
+        }
+    }
+}
+
+#change-password-page {
+    .content {
+        width: 400px;
+
+        p {
+            text-align: center
+        }
+
+        p.lead, h1 {
+            font-size: $font-size-large;
+            text-align: center;
+            margin-bottom: 20px;
+            font-weight: normal;
+        }
+        
+        .buttons {
+            &:not(.with-rows), .buttons-row {
+                display: flex;
+                margin-bottom: 10px;
+            }
+
+            &:not(.with-rows), .buttons-row:last-child {
+                margin-bottom: 20px;
+            }
+
+            button {
+                margin-right: 10px;
+
+                &:last-child {
+                    margin-right: 0;
+                }
+            }
+        }
+
+        .box-content {
+            margin-bottom: 10px;
+        }
+
+        .progress {
+            height: 15px;
+        }
```

Comment:
Seems like you are implementing alot of the stuff that is already implemented aboce. You added `#change-password-page` to the same list if with other frontend pages.